vhdbuilder/packer/install-dependencies.sh

```diff
@@ -24,6 +24,7 @@ source /home/packer/tool_installs_distro.sh
 CPU_ARCH=$(getCPUArch)  #amd64 or arm64
 VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
 COMPONENTS_FILEPATH=/opt/azure/components.json
+COMPONENTS_FILEPATH=/opt/azure/gpu-components.json
```

Comment:
does `gpu-components.json` already exist? 

also probably want to rename to GPU_COMPONENTS_FILEPATH as to not overwrite COMPONENTS_FILEPATH